install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -61,6 +61,12 @@ etcd3.example.com
 $ ansible-playbook [-i /path/to/file] \
```

Comment:
The example in the BZ was for `openshift-node`, but this was a close as I could find this this topic.